tests/queries/0_stateless/02479_race_condition_between_insert_and_droppin_mv.sh

```diff
@@ -42,7 +42,7 @@ TIMEOUT=55
 
 for i in {1..4}
 do
-    timeout $TIMEOUT bash -c drop_mv $i &
+    timeout $TIMEOUT bash -c "drop_mv $i" &
```

Comment:
I'm guessing the `insert 20` below has the same issue, but since the parameter isn't used at all in the function we can simply remove it.

Reply:
Yes, that true.
But I think that test would be broken after this fix. Since it had never done what supposed to be done.

Reply:
It seems like test works actually.
For the history: I have seen that the queries from that tests hung after all tests finished.